ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/byo-hosted-zone/provision/cucushift-installer-rehearse-gcp-ipi-xpn-byo-hosted-zone-provision-chain.yaml

```diff
@@ -1,13 +1,14 @@
 chain:
-  as: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-provision
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-byo-hosted-zone-provision
```

Comment:
In this chain, we are still using minimal-permission to run the installation, right ?